apps/vh/video-web/image-policy.yaml

```diff
@@ -16,6 +16,22 @@ spec:
 ---
 apiVersion: image.toolkit.fluxcd.io/v1beta1
 kind: ImagePolicy
+metadata:
+  name: vh-video-web-demo
+  annotations:
+    hmcts.github.com/prod-automated: disabled
+spec:
+  filterTags:
+    extract: $ts
+    pattern: '^dev-2122+-(?P<ts>[0-9]+)'
```

Comment:
you don't need the + after the PR number
should be `'^dev-2122-(?P<ts>[0-9]+)'`